nox.py

```diff
@@ -12,9 +12,100 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import os
+from __future__ import absolute_import
 
 import nox
+import os
+
+
+@nox.session
+@nox.parametrize('python_version', ['2.7', '3.4', '3.5', '3.6'])
```

Comment:
The google-cloud-python packages did 'python_version' -> 'py', 'system_tests' -> 'system', 'unit_tests' -> 'unit'.  Do you have any opinion on doing the same here?  I think I'm fine with either.

Reply:
Changed.